src/runtime/virtcontainers/fc.go

```diff
@@ -611,19 +599,29 @@ func (fc *firecracker) fcSetLogger() error {
 		return fmt.Errorf("Failed setting log: %s", err)
 	}
 
-	// listen to metrics file and transfer error info
+        fc.fcConfig.Logger = &models.Logger{
```

Comment:
This file is not formatted?

Reply:
hm ok, I force pushed after running gofmt